ddtrace/contrib/graphql/patch.py

```diff
@@ -0,0 +1,84 @@
+from typing import Union
+
+import graphql
+from graphql.language import Source
+
+from ddtrace import Span
+from ddtrace import config
+from ddtrace.constants import ANALYTICS_SAMPLE_RATE_KEY
+from ddtrace.constants import SPAN_MEASURED_KEY
+from ddtrace.internal.utils import get_argument_value
+from ddtrace.internal.utils.wrappers import unwrap as _u
+from ddtrace.pin import Pin
+from ddtrace.vendor.wrapt import wrap_function_wrapper as _w
+
+from .. import trace_utils
+from ...ext import SpanTypes
+
+
+config._add("graphql", dict(_default_service="graphql"))
+
+graphql_version_str = getattr(graphql, "version", "0.0.0")
```

Comment:
is `graphql.version_info` available back to 1.1? might make this easier for you?

Reply:
Yup it's available. I'll use that instead